lib/layout/box_layout.dart

```diff
@@ -219,21 +220,9 @@ class BoxLayoutState extends WidgetState<BoxLayout> with TemplatedWidgetState {
         IntrinsicHeight(child: boxWidget);
     }
 
-    // html-renderer has terrible scrolling performance when clipping is on, so disable it for now
-    Clip clipBehavior = Clip.hardEdge;
-    if (kIsWeb && isHtmlRenderer) {
-      clipBehavior = Clip.none;
-    }
-
-    Widget rtn = Container(
-        width: widget._controller.width != null ? widget._controller.width!.toDouble() : null,
-        height: widget._controller.height != null ? widget._controller.height!.toDouble() : null,
-        margin: widget._controller.margin,
-
-        clipBehavior: clipBehavior,
-        decoration: _buildBoxDecoration(),
-
-        child: flutter.InkWell(
+    Widget rtn = BoxWrapper(
```

Comment:
A box should only be clickable inside its border and not on its margin. You need to tell BoxWrapper to not include margin, then wrap the box inside your InkWell, then add the margin at the end. Look at how we did Image.